### DISCUSSION TITLE
Creating hyper 63 projects

### DISCUSSION THREAD

**Discussion by Benanna2019** [hyper63/atlas#73] [Q&A] [answered]
Should I create one hyper63 project for all my marketing api’s?

**Comment by twilson63** [comment]
It depends, when you say marketing apis what do you mean?

**Reply by Benanna2019** [reply]
So I am primarily referring to the etl portion of a hyper63 project. Say I am trying to get data from different marketing sources that I use (snapchat, Spotify, etc). Can I have that all run on one project? I guess this is more of a best practices question for using hyper63 when pulling from multiple data sources.

**Comment by twilson63** [comment, marked as answer]
Building an ETL process and using hyper63 as the data warehouse is one of many use cases, for an example of a service etl template 

Check out:

https://github.com/hyper63/etl-template2
